frontend/package.json

```diff
@@ -44,6 +45,7 @@
     "reselect": "^4.0.0",
     "sass": "^1.23.3",
     "seamless-immutable": "^7.1.4",
+    "semantic-ui-react": "^0.88.2",
```

Comment:
We're already using bootstrap -- can we not add the entire semantic-ui library? Could we instead have a component for a single search?

Reply:
If we can also change the font to the default font (`"Roboto", "Helvetica Neue", sans-serif`) that would be great!

Reply:
Got it! Let me see how to only use a single component.

Reply:
Even if we are adding the entire library, we are only importing the relevant js files from the library. Does this still matter?

Reply:
@Levilian did you get a chance to look into this?

Reply:
@epicfaace So it seems there are two issues. One is adding the entire semantic-ui-react package. The other is the potential clash between semantic css and material css. 

The issue I run into with the first one is that the code not only depends on the behavior of Search js in semantic ui, but also the behavior of, say, Textbox as well. What is the true impact of importing only relevant js files from a sizable package? I thought only the size of the imported files determines the load time? 

The second issue - I was able to identify the relevant parts of the semantic ui css that would maintain the current look. Do I simply copy and paste those parts into our already bulky css? Or is the goal to scrape the semantic ui css and build new css?